tests/test_icon_score_deployer.py

```diff
@@ -132,6 +133,34 @@ def test_deploy_when_score_depth_is_different(self):
             score_path: str = get_score_path(self.score_root_path, address)
             remove_path(score_path)
 
+    def test_deploy_replace_once(self):
+        """
+        Test for replacing the first occurrence only of the path which is upper than package.json
+        by using count on the function `replace`. It is supported on revision_3_or_more.
+        """
+        revision_list = [2, 3, 4]
```

Comment:
Please remove the revision 4 since we don't use it yet.

Reply:
It has been applied.